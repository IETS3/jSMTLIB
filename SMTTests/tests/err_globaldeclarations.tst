; checks global declarations
(set-option :global-declarations true)
(set-logic QF_UF)
(declare-fun a () Bool)
(reset-assertions)
(declare-fun a () Bool)  ; duplicate - ERROR
