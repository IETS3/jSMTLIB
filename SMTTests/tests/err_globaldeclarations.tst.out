success
success
success
success
(declare-fun a () Bool)  ; duplicate - ERROR
             ^
(error "Symbol a is already defined")
